classes/02class/exercises/c02-network08/ANSWER.md

```diff
@@ -30,4 +30,4 @@ Add your commands and their outputs here
 
 <!-- Don't change anything below this point-->
 ***
-Answer for exercise [c02-network08](https://github.com/devopsacademyau/academy/blob/893381c6f0b69434d9e8597d3d4b1c17f9bc1371/classes/02class/exercises/c02-network08/README.md)
\ No newline at end of file
+Answer for exercise [c02-network08](https://github.com/devopsacademyau/academy/blob/80a940b39bc3ae40378abe7af015cb3c207463f6/classes/02class/exercises/c02-network08/README.md)
```

Comment:
sorry to meddle, but maybe the link below is more precise? https://github.com/devopsacademyau/academy/blob/80a940b39bc3ae40378abe7af015cb3c207463f6/classes/02class/exercises/c02-network08/ANSWER.md

Reply:
I think the link is for the questions of the exercise, not the answers (the files that is being edited).